<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1143</width>
         <height>825</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Business Page:</string>
       </attribute>
       <widget class="QGroupBox" name="stateGroup">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>501</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Select Location</string>
        </property>
        <widget class="QComboBox" name="stateList">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>161</width>
           <height>22</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Lucida Sans</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>State</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>City</string>
         </property>
        </widget>
        <widget class="QListWidget" name="cityList">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>50</y>
           <width>131</width>
           <height>151</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QListWidget" name="zipcodeList">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>50</y>
           <width>161</width>
           <height>151</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>30</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Zipcode</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="businesscategoriesGroup">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>1141</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>Filter On Categories</string>
        </property>
        <widget class="QListWidget" name="businessTableCategory">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>231</width>
           <height>201</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Lucida Sans</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Category</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="businessTableDetails">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>60</y>
           <width>791</width>
           <height>201</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>50</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>30</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Businesses</string>
         </property>
        </widget>
        <widget class="QToolButton" name="searchButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>60</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/search.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QToolButton" name="clearButton">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>110</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/clear.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="ZipStatistics">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>0</y>
          <width>631</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(181, 206, 227);</string>
        </property>
        <property name="title">
         <string>ZipCode Statistics</string>
        </property>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>131</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string># of Businesses:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>90</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Total Population:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>140</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Average Income:</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="topCategories">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>40</y>
           <width>271</width>
           <height>171</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>20</y>
           <width>141</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Top Categories</string>
         </property>
        </widget>
        <widget class="QLabel" name="nobusslabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/analytics.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="totalpoplabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/population.jpeg);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="avgincomelabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>41</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/income.png);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="NumberOfBusiness">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>40</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="TotalPopulation">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>90</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="AverageIncome">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>140</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0,0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="PopBusinessesgroup">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>530</y>
          <width>541</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(255, 202, 198);</string>
        </property>
        <property name="title">
         <string>Popular Businesses (in zipcode)</string>
        </property>
        <widget class="QTableWidget" name="popularBusinessTable">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>491</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QGroupBox" name="SussBussinessesgroup">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>530</y>
          <width>541</width>
          <height>341</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;Lucida Sans&quot;;
background-color: rgb(203, 255, 199);</string>
        </property>
        <property name="title">
         <string>Successful Businesses (in zipcode)</string>
        </property>
        <widget class="QTableWidget" name="sucessfulBusiness">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>491</width>
           <height>301</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
font: 14pt &quot;Lucida Sans&quot;;</string>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
       <widget class="QToolButton" name="refreshButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>530</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
image: url(/Users/aa/Documents/Cpts 451/Milestone-3/Icons/refresh.png);
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1167</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
